content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -105,6 +105,7 @@ different Kubernetes components.
 | `HPAScaleToZero` | `false` | Alpha | 1.16 | |
 | `HugePageStorageMediumSize` | `false` | Alpha | 1.18 | |
 | `HyperVContainer` | `false` | Alpha | 1.10 | |
+| `ImmutableEphemeralVolumes` | `false` | Alpha | 1.18 | |
```

Comment:
@wojtek-t , If time, you probably want to add a brief description about this feature.
See the `feature-gates` section on this page, https://deploy-preview-19297--kubernetes-io-vnext-staging.netlify.com/docs/reference/command-line-tools-reference/feature-gates/#feature-gates

Reply:
done